completion/ctex.cwl

```diff
@@ -0,0 +1,278 @@
+# ctex package
+# Darcy Hu <hot123tea123@gmail.com> 2016
+#modified zepinglee 26 Jan 2021
+
+#include:zhnumber
+
+## Sec. 4.3
+\songti
+\heiti
+\fangsong
+\kaishu
+\lishu#*
+\youyuan#*
+\yahei#*
+\pingfang#*
+
+\ctexset{keyvals}
+#keyvals:\ctexset
+fontset=#adobe,fandol,founder,mac,macnew,macold,ubuntu,windows,none
+#
+# Sec. 5.3
+punct=#quanjiao,banjiao,kaiming,CCT,plain
+space=#true,false,auto
+autoindent=
+linestretch=#*
+#
+# Sec. 6.1
+today=#small,big,old
+#
+# Sec. 6.2
+contentsname=
+listfigurename=
+listtablename=
+figurename=
+tablename=
+abstractname=
+indexname=
+appendixname=
+bibname=
+proofname=
+refname=
+algorithmname=
+continuation=
+#
+# Sec. 7.1
+part=
+chapter=
+section=
+subsection=
+subsubsection=
+paragraph=
+subparagraph=
+#
+# Sec. 7.1
+part/numbering=#true,false
```

Comment:
```suggestion
part/numbering#true,false
```
This will allow input `part/numbering`, which makes use of the default value (usually `true`). See https://github.com/texstudio-org/texstudio/pull/687#issuecomment-520205897.

Reply:
The explicit “`=true/false`” pattern is used in the documentation thus i suppose it should be encouraged.